scripts/prow-smoke-test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+
+# Allow podman or docker 
+alias docker='podman'
+
+# Check if a container engine runtime is installed
+hash podman 2>/dev/null || { echo >&2 "Container engine not installed. Install podman or docker and run the script again"; exit 1; }
+
+# Check if any parameters are passed
+if [ $# -eq 0 ]; then
+    echo "🙉 Run the script passing 'validate' or 'preview' params, for example:"
+    echo ""
+    echo "./scripts/prow-smoke-test.sh validate"
+    exit 1
+fi
+
+TEST=$1
+DISTRO=$2
+CONTAINER_IMAGE=quay.io/redhat-docs/openshift-docs-asciidoc
+
+if [[ -z "$DISTRO" && "$TEST" == "preview" ]]; then
+    DISTRO="openshift-enterprise"
+    echo ""
+    echo "🚧 Building with openshift-enterprise distro..."
+    echo "💡 To build a specific distro, pass a second \$DISTRO param. For example:"
+    echo ""
+    echo "./scripts/prow-smoke-test.sh preview openshift-rosa"
+    echo ""
+elif [[ -n "$DISTRO" && "$TEST" == "preview" ]]; then
+   echo "Building $2 distro..."
+fi
+
+CONTAINER_WORKDIR=$(podman run --rm $CONTAINER_IMAGE /bin/bash -c 'echo $PWD')
+
+if [[ "$TEST" == "preview" ]]; then
+    podman run --rm -it -v "$(pwd)":${CONTAINER_WORKDIR}:Z $CONTAINER_IMAGE asciibinder build -d "$DISTRO"
+elif [[ "$TEST" == "validate" ]]; then
+  podman run --rm -it -v "$(pwd)":${CONTAINER_WORKDIR}:Z $CONTAINER_IMAGE sh -c 'scripts/check-asciidoctor-build.sh && python3 build_for_portal.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.15 --no-upstream-fetch && python3 makeBuild.py'
```

Comment:
**suggestion:** We should also add variables for product name and version. That way I can use it to validate ACS and other distros.

Reply:
If we do that we must also update the example usage.

Reply:
Reworked this completely to support the various distros. We now set defaults for $PRODUCT_NAME, $VERSION, $DISTRO and these can be overridden in the CLI.